src/Take.Elephant.Redis/MapBase.cs

```diff
@@ -32,21 +32,31 @@ protected MapBase(string mapName, IConnectionMultiplexer connectionMultiplexer,
         public abstract Task<bool> TryRemoveAsync(TKey key, CancellationToken cancellationToken = default);
 
         public abstract Task<bool> ContainsKeyAsync(TKey key, CancellationToken cancellationToken = default);
-        
+
         #region IExpirableKeyMap<TKey, TValue> Members
 
+        protected virtual Task<bool> SetRelativeKeyExpirationAsync(RedisKey key, TimeSpan ttl)
```

Comment:
Add `async/await` instead of returning a `Task` directly.

Reply:
I did not use async/await here for the same reason as explained in the [comment below](https://github.com/takenet/elephant/pull/58#pullrequestreview-915741691).